docker/Dockerfile

```diff
@@ -19,3 +15,13 @@ RUN chmod +x /wait
 
 RUN pip install --no-cache-dir -r requirements/dev.txt \
     && rm -rf /tmp/*
+
+COPY . .
+
+EXPOSE 5000
+CMD ["flask", "run", "--host=0.0.0.0"]
```

Comment:
I see `--host` can also be passed as [`SERVER_NAME`](https://flask.palletsprojects.com/en/1.1.x/api/?highlight=flask%20run#flask.Flask.run) which would allow to override the `CMD` without having to remember to also pass `--host`. But:
- I don't know how to pass `SERVER_NAME` in
- this is a `dev` image
so good as-is.